__init__.py

```diff
@@ -133,8 +133,21 @@ def currency_filter(num, grouping=True):
 def date_filter_sm(date_str):
     if not date_str:
         return ''
-    return parse_date(date_str).strftime('%m%y')
+    return parse_date(date_str).strftime('%m/%y')
 
+@app.template_filter('date_md')
```

Comment:
Both newly added filters need unit tests. Tests can be added to `openfecwebapp/tests/test_utils.py` after #167 is merged.